stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -63,7 +66,12 @@ private Pattern getWordBoundaryPattern() {
     if (mViewProperties == null) {
       synchronized (this) {
         if (mViewProperties == null) {
-          List<ViewCSSProperty> props = new ArrayList<>();
+          Set<ViewCSSProperty> props = new TreeSet<>(new Comparator<ViewCSSProperty>() {
```

Comment:
What about keeping it as a `List` and then sorting afterward? We're intentionally avoiding things like `Map` and `Set` because they add an extra allocation per entry. This code also adds an extra copy operation (`new ArrayList<>(props)') that we'd prefer to avoid.